lib/execjs/external_runtime.rb

```diff
@@ -28,8 +28,15 @@ def exec(source, options = {})
         source = @runtime.compile_source(source)
 
         tmpfile = write_to_tempfile(source)
+
+        if ExecJS.cygwin?
+          filepath = `cygpath -m #{tmpfile.path}`.rstrip
```

Comment:
**Update**
Even if it's working with `-w`, path generated is (*note the backslashes*):
`C:\CYGWIN~1\path/to/something`

And that a bit ugly, while `-m` generate that:
`C:/CYGWIN~1/path/to/something`

What do you think?